python/hsfs/feature_group.py

```diff
@@ -535,6 +537,35 @@ def save_validation_report(
             return self._validation_report_engine.save(report).to_ge_type()
         return self._validation_report_engine.save(report)
 
+    def get_validation_history(
+        self,
+        expectation_id: int,
+        sort_by: Optional[str] = "validation_time:desc",
+        filter_by: Optional[str] = None,
+        offset: Optional[int] = None,
+        limit: Optional[int] = None,
+    ) -> List[ValidationResult]:
+        """Fetch validation history of an Expectation specified by its id.
+
+        # Arguments
+            expectation_id: id of the Expectation for which to fetch the validation history
+            sort_by: sort the validation results according to validation time, descending or ascending. Value can be either
+                validation_time:desc or validation_time:asc
+            filter_by: filter the validation results based on validation time or ingestion result.
```

Comment:
How about hiding the implementation detail of the backend? We can have
```
ingested_only=False
rejected_only=False
# For the time we can reference to fg.read_changes
start_validation_time: Union[str, int, datetime, date],
end_validation_time: Union[str, int, datetime, date],
```
This seems more user friendly.